.github/workflows/package_for_test.yml

```diff
@@ -98,14 +98,28 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: Neuron-Mac
-          path: release/Neuron-*-mac.zip
+          path: release/Neuron-*-x64-mac.zip
+
+      - name: Upload Neuron App Zip(arm64)
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v2
+        with:
+          name: Neuron-Mac
+          path: release/Neuron-*-arm64-mac.zip
```

Comment:
`path` should be updated as https://github.com/nervosnetwork/neuron/pull/2515/files#diff-170ebc8e4dc40acf23cbe0ecce5f3e2aef1652511f59860db704106b197e1d52R113